packages/ndla-ui/src/TopicIntroductionList/TopicIntroduction.tsx

```diff
@@ -7,37 +7,49 @@
  */
 
 import React from 'react';
-import PropTypes from 'prop-types';
 import BEMHelper from 'react-bem-helper';
+import { useTranslation } from 'react-i18next';
 import { Additional, Core } from '@ndla/icons/common';
 import Tooltip from '@ndla/tooltip';
 import SafeLink from '@ndla/safelink';
-import { TopicShape, ShortcutShape } from '../shapes';
 import TopicIntroductionShortcuts from './TopicIntroductionShortcuts';
+import { Shortcut, Topic } from './TopicIntroductionList';
 
 const topicClasses = new BEMHelper({
   prefix: 'c-',
   name: 'topic-introduction',
   outputIsString: true,
 });
 
+interface Props {
+  additional?: boolean;
+  showAdditionalCores?: boolean;
+  topic: Topic;
+  toTopic: Function;
+  subjectPage?: boolean;
+  shortcuts?: Shortcut[];
+  twoColumns?: boolean;
+  shortcutAlwaysExpanded?: boolean;
+  id: string;
+}
+
 export const TopicIntroduction = ({
   toTopic,
   topic,
   subjectPage,
   shortcuts,
-  messages,
```

Comment:
Er det for å forenkle koden du fjerner messages? For det er vel ikkje et problem i seg sjølv at dei defineres utenfor komponenten. Det er kanskje ryddigast slik?

Reply:
`messages` blir passert nedover fra `TopicIntroductionList`, ikke fra en frontend. Jeg synes det er mindre oversiktlig med messages enn det er å faktisk bruke `t('r.l') direkte, da jeg slipper å hoppe mellom filer.